src/wombats/constants.clj

```diff
@@ -6,3 +6,28 @@
 (defonce github-scopes "user:email")
 (defonce github-user-profile-url "https://api.github.com/user")
 (defonce github-repo-api-base "https://api.github.com/repos/")
+
+;; Error Codes
+;; Format: a-bb-ccc
+;; a: File type: 0 handlers
+;;               1 dao
+;; b: File number: Increments by one for each file
+;; c: Error number: Increments by one for each error
+(def errors
```

Comment:
I think this is fine for now, and could possible be addressed in a future PR. I'd be more of a fan of keyword error codes where the first parts says whether it's a handler or dao, then it's the filename, and lastly a descriptor to the error code (number could also work there if you wanted).

Example:
`:dao_finalizer_arena-not-found` or
`:dao-finalizer-0`